.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -0,0 +1,27 @@
+---
+name: Feature request
+about: Submit a feature request
+title: "[FR] - "
```

Comment:
Do we need the title prefixes if we have the labels?

Reply:
No, I suppose not